libraries/AP_Math/control.cpp

```diff
@@ -324,6 +324,38 @@ void shape_pos_vel_accel_xy(const Vector2p& pos_input, const Vector2f& vel_input
     shape_vel_accel_xy(vel_target, accel_input, vel, accel, accel_max, jerk_max, dt, limit_total_accel);
 }
 
+// limit_accel_xy limits the acceleration to prioritise acceleration perpendicular to the provided velocity vector.
```

Comment:
It would be good to add a comment saying when it returns true vs false.  I guess it is true if the passed in accel argument has been limited, false if it has been left unchanged.